src/platform/CHANGELOG.md

```diff
@@ -14,7 +14,7 @@ CHANGELOG
    - AWS Bedrock (Anthropic Claude, Meta Llama, Amazon Nova)
    - Mistral AI (language models and embeddings)
    - Meta Llama (via Azure, Ollama, Replicate, AWS Bedrock)
-   - Ollama (local model hosting)
+   - Ollama (local model hosting and embeddings)
```

Comment:
I don't think this addition is meaningful, since we create the embeddings also with models and the main feature of ollama is running models locally